src/types/Calculator.ts

```diff
@@ -395,6 +400,18 @@ export interface CalculationOutput<T extends CalculationOptions> extends Calcula
   parameters?: { [key: string]: any };
   coverageHTML?: string;
   groupClauseCoverageHTML?: Record<string, string>;
+  groupClauseUncoverageHTML?: Record<string, string>;
+  groupClauseCoverageDetails?: Record<string, ClauseCoverageDetails>;
+}
+
+export interface ClauseCoverageDetails {
+  totalClauses: number;
+  coveredClauses: number;
+  uncoveredClauses: {
```

Comment:
Because the other two clauses details are numbers, this naming seems a a bit incongruous. It would like me to expect all of them to be the same type. Maybe we change the other two to `totalClausCount` and `coveredClauseCount` or something like that so that it doesn't lead to the expectation that `uncoveredClauses` also has a count?

Reply:
Renamed in ca98110 and added uncovered count